src/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Movies</title>
+</head>
+<body>
+  <header>
+    <nav class="flex">      
+      <ul class="flex">
+        <li id="logo"><a href="index.html">Movie site logo</a></li>
+        <li id="all-movies">All Movies</li>
+        <li>Categories</li>
+        <li>Popular Movies</li>
+      </ul>
+    </nav>
```

Comment:
[OPTIONAL] 👍🏽 

- I think it will be better if you removed the cursor: pointer from the nav bar since they are not clickable links. 🤗